examples/remix-with-typescript/app/styles/dark.css

```diff
@@ -0,0 +1,7 @@
+:root {
+  --color-foreground: hsl(0, 0%, 100%);
+  --color-background: hsl(0, 0%, 7%);
+  --color-links: hsl(213, 100%, 73%);
+  --color-links-hover: hsl(213, 100%, 80%);
+  --color-border: hsl(0, 0%, 25%);
+}
```

Comment:
Why do we have this CSS file and the others? If the styles are really needed, I think that they should use the system.
```suggestion
```

Reply:
Let me remove these and add the same content as in the other templates. What do you think?